scripts/supercloud/run_pg3_tamp_experiments.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+FILE="scripts/supercloud/submit_supercloud_job.py"
+ALL_NUM_TRAIN_TASKS=(
+     "10"
+     "50"
+     "100"
+)
+ENVS=(
+    "cover"
+    "painting"
+    "screws"
+    "repeated_nextto"
+    "cluttered_table"
+    "coffee"
+)
+
+for ENV in ${ENVS[@]}; do
+  for NUM_TRAIN_TASKS in ${ALL_NUM_TRAIN_TASKS[@]}; do
+
+      if [ $ENV = "painting" ] || [ $ENV = "cluttered_table" ] || \
+          [ $ENV = "coffee" ] then
+        COMMON_ARGS="--env $ENV  --strips_learner oracle \
+              --sampler_learner oracle --num_train_tasks $NUM_TRAIN_TASKS \
+              --pg3_hc_enforced_depth 1"
+      else
+        COMMON_ARGS="--env $ENV  --strips_learner oracle \
+              --sampler_learner oracle --num_train_tasks $NUM_TRAIN_TASKS \"
+      fi
+
+      # If in painting enviroment override common arguments and included options
+      if [ $ENV = "painting" ]; then
+        INCLUDED_OPTIONS="--painting_lid_open_prob 1.0"
```

Comment:
we can actually just add this flag and all of the env-specific ones to COMMON_ARGS. if we use the flag `--painting_lid_open_prob` while using the cover env or whatever it will just have no effect, which is fine. this should simplify this bash file a bit

Reply:
Oh, good to know! Thanks for the tip